src/codeu/chat/server/View.java

```diff
@@ -77,7 +80,10 @@ public View(Model model) {
 
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
-
+    
+  @Override
+  public ServerInfo getInfo() { return info; }
```

Comment:
I don't think this method is actually called by your code. There's already a `getInfo` method in Context.java that's already used. You should remove this method since it's unused.